README.md

```diff
@@ -207,4 +207,23 @@ see the [Scaling Lightning](https://github.com/scaling-lightning/scaling-lightni
 project.
 
 ## Docker
-If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
\ No newline at end of file
+If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
+
+## Contributing
+SimLN is open and welcomes contribution from the community. When creating a pull request, ensure that you adhere to the following:
```

Comment:
Could you add in a note that you can ask questions on [discussions](https://github.com/bitcoin-dev-project/sim-ln/discussions/123) if you don't know where to start? Trying to push people towards that as a support forum.